File: JS/rental-app-js/models/Rental.js
class Rental {
    constructor(item, startDate, endDate) {
      this.item = item;
      this.startDate = new Date(startDate);
      this.endDate = new Date(endDate);
    }
  
    isActive() {
      return this.endDate >= new Date();
    }
  }
  
  module.exports = Rental;
  

File: JAVA/rental-app-java/src/models/Rental.java
package models;

import java.time.LocalDate;

public class Rental {
    private Item item;
    private LocalDate startDate;
    private LocalDate endDate;

    public Rental(Item item, LocalDate startDate, LocalDate endDate) {
        this.item = item;
        this.startDate = startDate;
        this.endDate = endDate;
    }

    public Item getItem() {
        return item;
    }

    public LocalDate getStartDate() {
        return startDate;
    }

    public LocalDate getEndDate() {
        return endDate;
    }

    public boolean isActive() {
        return endDate.isAfter(LocalDate.now());
    }
}
